### PR TITLE
Create export script and configs

### PR DESCRIPTION
**Pull request by BenWu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,57 @@
...
 
-The purpose of this job is to continuously backfill past days over time.
-Past Play Store data has been found to still update over time
-(e.g. data from a day two weeks ago can still be updated)
+The purpose of this job is to be scheduled to run regularly in order to continuously backfill past days over time.
+Past Play Store data has been found to still update over time (e.g. data from a day two weeks ago can still be updated)
 so regular backfills of at least 30 days are required.
-The BigQuery Play Store transfer job has a non-configurable refresh
-window size of 7 days which is insufficient.
+The BigQuery Play Store transfer job has a non-configurable refresh window size of 7 days which is insufficient.
```
Comment (fbertsch) [inline README.md:9]: This is wild to me. So basically the BQ Play store transfer job is unusable.

File: play_store_export/cancel_transfers.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            if delete_count % 20 == 0:  # pause to avoid hitting api rate limit
+                time.sleep(3)
+            run_date = datetime.datetime.utcfromtimestamp(transfer_run.run_time.seconds).date()
+            client.delete_transfer_run(transfer_run.name)
+            delete_count += 1
+            print(f"Cancelled {transfer_run.name} for date {run_date}")
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```
Comment (fbertsch) [inline play_store_export/cancel_transfers.py:31]: Click might be a good alternative in the future. No need to change it here though.

File: play_store_export/export.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    else:
+        result = "unspecified"
+
+    logging.info(f"Transfer for {run_date} {result}")
+
+    return state
+
+
+def start_export(project: str, transfer_config_name: str, transfer_location: str,
+                 base_date: datetime.date, backfill_day_count: int = 35):
```
Comment (fbertsch) [inline play_store_export/export.py]: nit: 35 is defined here and below. I would remove it from here and let it solely be defined by the argparser.

**Review by fbertsch — APPROVED** [review]
This is really nice! Well done! Happy to see that the Python API for the transfer stuff is getting better as well.

File: README.md (hunk 1 above)
Comment (BenWu) [reply README.md:9]: From what I understand this is only an issue with the retention metrics which are forward looking up to 30 days.  I'll add that note to the readme

**Pull request merged**